client/src/components/Map/Map.js

```diff
@@ -1,14 +1,27 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types';
 import uniq from 'lodash/uniq';
+import { createSelector } from 'reselect';
+import cookies from 'browser-cookies';
```

Comment:
Cookies are sent with every request to the server, but we do not need this. Use `LocalStorage` instead for this task.

Reply:
Good point. I thought you could just store cookies locally, but I was wrong.  I'll change it to use local storage instead.